Makefile

```diff
@@ -14,6 +14,34 @@ export HTMLTEST_LOG_LEVEL ?= 2
 deps: packages/install/hugo \
 	  packages/install/htmltest
 
+# Install useful atom plugins
+deps/atom:
+	@apm install -s \
+		modular-snippets \
+		language-hugo \
+		autocomplete-paths \
+		markdown-badges-snippets \
+		editorconfig \
+		linter-ui-default \
+		linter-markdown \
+		language-markdown \
+		markdown-table-editor \
+		markdown-writer \
+		tool-bar-markdown-writer \
+		markdown-toc \
+		project-manager \
+		local-config \
+		autocomplete-paths
+
+	@echo "Please follow that steps on first run"
```

Comment:
`Now complete the setup by performing the following tasks:`

Reply:
Done